file_reader.py

```diff
@@ -0,0 +1,28 @@
+import os
+
+
+class FileReader:
+    def __init__(self, directory):
+        self.directory = directory
+
+    def read_weather_file(self, filepath):
```

Comment:
Add docString to the method and read the files as csv. Read the first row as header, this will improve readability and ease in indexing to use this data.